packages/perseus/src/widgets/label-image/validate-label-image.test.ts

```diff
@@ -0,0 +1,29 @@
+import validateLabelImage from "./validate-label-image";
+
+import type {PerseusLabelImageUserInput} from "../../validation.types";
+
+describe("scoreLabelImage", function () {
```

Comment:
Micro nit that's more personal taste than anything and I know it's just copied from the original, but in this context I prefer arrow functions.

```suggestion
describe("scoreLabelImage", () => {
```

Reply:
Fixed :)